arlo-client/src/components/utilities.ts

```diff
@@ -104,3 +111,16 @@ export const openQR = (id: string, name: string) => {
     )
   }
 }
+
+export const toaster = (response: IErrorResponse): boolean => {
```

Comment:
Rather than requiring `response` to be `IErrorResponse`, this seems to be answering the question "is `response` an error response?". So from a type system perspective, this might be better:

```ts
const getErrorsFromResponse = (
  response: unknown
): { message: string }[] | undefined => {
  if (typeof response !== 'object' || !response) {
    return
  }

  const errors = (response as { [key: string]: unknown })['errors']

  if (!Array.isArray(errors)) {
    return
  }

  return errors
}

export const toaster = (response: object): response is IErrorResponse => {
  const errors = getErrorsFromResponse(response)
  if (errors) {
    toast.error(
      'There was a server error regarding: ' +
        errors.map(({ message }) => message).join(', ')
    )
    return true
  } else {
    return false
  }
}
```